src/EdFi.Admin.LearningStandards.Core/Models/FromCsv/LearningStandardMetaData.cs

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using System.Collections.Generic;
+using System.Linq;
+
+namespace EdFi.Admin.LearningStandards.Core.Models.FromCsv
+{
+    public class LearningStandardMetaData
+    {
+        public string Name { get; set; }
+        public string DataType { get; set; }
```

Comment:
If these need to be strings, I would try to have a set of string constants for the known data-types so far, like "array". If storage type doesn't matter, perhaps an actual `Type`, or `enum` instead? I think there are some type conversions below; might help a bit? Just thinking out loud.